typescript.js

```diff
@@ -49,6 +49,10 @@ module.exports = {
         ],
         '@typescript-eslint/prefer-reduce-type-parameter': 'warn',
         '@typescript-eslint/prefer-return-this-type': 'warn',
+        '@typescript-eslint/restrict-template-expressions': [
+          'error',
+          { allowNumber: true },
+        ],
```

Comment:
I think adding [`{ allowBoolean: true }`](https://typescript-eslint.io/rules/restrict-template-expressions/#allowboolean) would also be reasonable :thinking: